eva-server/src/main/java/uk/ac/ebi/eva/server/Utils.java

```diff
@@ -18,13 +18,29 @@
  */
 package uk.ac.ebi.eva.server;
 
+import org.apache.commons.lang3.StringUtils;
 import org.opencb.datastore.core.QueryOptions;
 import org.springframework.data.domain.PageRequest;
 
 import uk.ac.ebi.eva.lib.repository.VariantEntityRepository;
 
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class Utils {
 
+    private static Map<String, String> apiToMongoDocNameMap;
+
+    static {
+        initApiToMongoDocNameMap();
+    }
+
+    public static Map<String, String> getApiToMongoDocNameMap() {
```

Comment:
Instead of this, a method like the following could be used to hide the fact that we use a Map and avoid the need to return an unmodifiable copy.

```java
public static String getDatabaseFieldMapping(String apiFieldName) {
    return apiToMongoDocNameMap.get(apiFieldName);
}
```

It would also be easier to test (see comment on `UtilsTest` class).